### ISSUE TITLE
get_l1_target 的疑惑

### ISSUE THREAD

**Issue by EighteenSprings** [Megvii-BaseDetection/YOLOX#1029]
https://github.com/Megvii-BaseDetection/YOLOX/blob/d669fd38d9ac672a9e5a7eab5aa61d3be6b3959a/yolox/models/yolo_head.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L417

我们目标应该是求 outputs 和 target 框的 l1， 但是这里为什么需要将 target 框的 box 信息除以 stride 再 log？ 难道前面 outputs 的结果没有处理么，在 forward 过程中，我们得到的 outputs， 是通过 get_output_and_grid 得到的，其中包含映射会输入尺寸的操作
```python
output[..., :2] = (output[..., :2] + grid) * stride
output[..., 2:4] = torch.exp(output[..., 2:4]) * stride
```
我感觉写得是正确的，但是不理解这一步的关键所在，可以麻烦解释下么

**Comment by EighteenSprings** [comment]
最后是求 origin_preds 和 target 的 loss 所以需要这样处理
